### DISCUSSION TITLE
Github Page Error: Page Build Failed

### DISCUSSION THREAD

**Discussion by makecent** [mmistakes/minimal-mistakes#3081] [Q&A] [answered]
## Environment

- Minimal Mistakes version: 4.24.0
- Ruby gem or remote theme version: N/A
- Jekyll version: N/A
- Git repository URL: https://github.com/makecent/makecent.github.io
- GitHub Pages hosted (if yes provide URL to site): Failed
- Operating system: N/A

## Reproduce

When I clone my repository and run ``bundle exec jekyll serve``, It worked well. But in Github Pages settings, the error below is displayed:
![image](https://user-images.githubusercontent.com/42603768/126902333-1833ef55-078d-4564-896d-a2294baf5bb4.png)

I include 
```yaml
gem "jekyll-remote-theme"
gem "jekyll-include-cache", group: :jekyll_plugins
```
in my [Gemfile](https://github.com/makecent/makecent.github.io/blob/bddf078110713d98879e9e4577dfc24f2e6e59d8/Gemfile#L1-L10) and added corresponding codes:
```yaml
plugins:
  - jekyll-paginate
  - jekyll-sitemap
  - jekyll-gist
  - jekyll-feed
  - jekyll-include-cache
  - jekyll-remote-theme
```
into the [_config.yml](https://github.com/makecent/makecent.github.io/blob/bddf078110713d98879e9e4577dfc24f2e6e59d8/_config.yml#L211-L217). Problems still exist.

**Comment by iBug** [comment]
![image](https://user-images.githubusercontent.com/7273074/126911402-b11b5c43-3fe0-4e9c-a704-18ca57948cf6.png)

<https://github.com/makecent/makecent.github.io/blob/bddf078110713d98879e9e4577dfc24f2e6e59d8/_config.yml#L11>

**Reply by makecent** [reply]
Sorry, I can't get your meaning

**Comment by makecent** [comment, marked as answer]
I resolved it by correcting the missing **"** in ``url``  in ``_config.yml``
